elide-core/src/main/java/com/yahoo/elide/Elide.java

```diff
@@ -119,13 +138,19 @@ protected Elide(AuditLogger auditLogger,
                   DataStore dataStore,
                   EntityDictionary dictionary,
                   JsonApiMapper mapper,
-                  Function<RequestScope, PermissionExecutor> permissionExecutor) {
+                  Function<RequestScope, PermissionExecutor> permissionExecutor,
+                  List<JoinFilterStrategy> joinFilterStrategies,
+                  List<SubqueryFilterStrategy> subqueryFilterStrategies,
+                  boolean useFilterExpressions) {
```

Comment:
This is a breaking change because sub-classes could be depending on this constructor.